compass/ocean/tests/tides/forward/namelist.ocean

```diff
@@ -4,10 +4,14 @@ config_calendar_type = 'noleap'
 config_pio_num_iotasks = 0
 config_pio_stride = 1
 config_time_integrator = 'RK4'
+config_use_mom_del2 = .true.
+config_mom_del2 = 3.333
+config_use_mom_del4 = .true.
```

Comment:
I didn't do a lot of tuning on the del^k dissipation, so this could be looked at in the future if there's interest.